dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -633,3 +633,19 @@ ssl.client.handshake.error.CWPKI0824E=CWPKI0824E: SSL HANDSHAKE FAILURE:  Host n
 ssl.client.handshake.error.CWPKI0824E.explanation=An error occurred during the SSL handshake.  Host name verification is enabled and server's identity can not be verified.  The host name used to access the server does not match the server certificate's SubjectDN or Subject Alternative Name information.
 ssl.client.handshake.error.CWPKI0824E.useraction=When host name verification is enable the server's being accessed needs to be setup with a certificate contain the proper information needed to verify the server's identity.
 # -------------------------------------------------------------------------------------------------
+ssl.cacerts.error.CWPKI0825W=CWPKI0825W: The trustDefaultCerts attribute is enabled and the default truststore failed to initialize. Only the configured truststore is used, and the default truststore is not used. The [{0}] SSL configuration is used without the default truststore. The exception that is returned is: [{1}].
+ssl.cacerts.error.CWPKI0825W.explanation=The default truststore was not able to be set up because a problem occurred. It is not used for trust. Only the configured SSL configuration is used.
```

Comment:
**Change**
The default truststore was not able to be set up because a problem occurred. It is not used for trust. Only the configured SSL configuration is used.
**to**
The default truststore did not initialize because an error occurred. The default truststore is not used for trust. Only the configured SSL configuration is used.